lib/preview/routes/index.js

```diff
@@ -30,16 +32,19 @@ const buildIndexHtml = async (sessionInfo) => {
       <body style="height:100%;display:flex;flex-direction:column;align-items:center;">
         <div>
           <h1>Domains</h1>
-          ${ domains ? getSiteList(domains, PORT) : '<p>No domains found</p>' }
+          ${ domains.length ?
+            getSiteList(domains, PORT) :
+            "<p>No domains found. You either don't have any domains set up in your portal or your personal access key is missing a scope 'cms.domains.read' required for this feature.</p>"
+          }
```

Comment:
We log the error to console if the scope is missing but good to put one here too.